src/main/kotlin/no/nav/tilleggsstonader/sak/statistikk/vedtak/VedtaksstatistikkDvh.kt

```diff
@@ -22,49 +26,74 @@ class VedtaksstatistikkDvh(
     val eksternBehandlingId: Long,
     val relatertBehandlingId: Long?, // Ekstern behandlingsid på relatert behandling
     val adressebeskyttelse: AdressebeskyttelseDvh,
-    val tidspunktVedtak: ZonedDateTime,
+    val tidspunktVedtak: LocalDateTime,
     val målgruppe: List<MålgruppeDvh>,
     val aktivitet: List<AktivitetDvh>,
-    val vilkårsvureringer: List<VilkårsvurderingDvh>,
+    val vilkårsvurderinger: List<VilkårsvurderingDvh>,
     val person: String,
     val barn: List<BarnDvh>,
     val behandlingType: BehandlingTypeDvh,
     val behandlingÅrsak: BehandlingÅrsakDvh,
     val vedtakResultat: VedtakResultatDvh,
-    val vedtaksperioder: VedtaksperioderDvh,
+    val vedtaksperioder: List<VedtaksperiodeDvh>,
     val utbetalinger: List<UtbetalingDvh>,
     val stønadstype: StønadstypeDvh = BARNETILSYN,
-    val kravMottatt: LocalDate,
+    val kravMottatt: LocalDate?,
     val årsakRevurdering: ÅrsakRevurderingDvh? = null,
     val avslagÅrsak: AvslagÅrsakDvh? = null,
 
-)
+    )
 
 enum class StønadstypeDvh {
     BARNETILSYN,
 }
 
 data class UtbetalingDvh(
-    val gyldigFom: LocalDate,
-    val gyldigTom: LocalDate,
+    val utbetalingsdato: LocalDate,
     val beløp: Int,
-    val satstype: SatstypeDvh = SatstypeDvh.DAGLIG,
-)
+) {
+    companion object {
+        fun fraDomene(ytelser: List<AndelTilkjentYtelse>) = ytelser.map {
+            UtbetalingDvh(utbetalingsdato = it.fom, beløp = it.beløp)
```

Comment:
Dette blir egentlig ikke korrekt, men det er ikke så lett å få en riktig dato 😕  (Meget mulig EF også gjør feil på dette) Lurer på om `tidspunktVedtak` egentlig er bedre og bare bruke det på alle.

Hvis det feks. 15, mai innvilges stønad for perioden 1. april-30. april vil tilkjenteYtelser bestå av en andel med `fom` 1. april. Men vedtak vil jo iverksettes 15. mai, og utbetales samme dag.

Reply:
Hvis de lurer på datoen som utbetalingen faktisk gjelder for så blir jo dette riktig, men om de lurer på når det ble iverksatt er det noe annet. Da vil vedtaksdato eller første i mnd være mer riktig